### ISSUE TITLE
Color palette in website

### ISSUE THREAD

**Issue by DanielleRameau** [sl-design-system/components#956]
**Color palette in documentation website**
The documentation website doesn't have the complete color palette of the SL theme. Please add so we solve contrast issues.

https://www.figma.com/file/DutPvhTyFEeW44CMatqSP6/Sanoma-Learning-Design-Tokens?type=design&node-id=1%3A175&mode=design&t=1ZyNeAWNOr6Dz07G-1

Tokens can be found in the base of the SL theme

```[tasklist]
### Tokens
- [ ] add SL color palette tokens to website so we can use the same tokens in the user interface
```

**Implement tokens in code block due to a11y issues**

```[tasklist]
### Code block in documentation pages
- [x] Change token.property to: #2351DB (accent.500)
- [x] Change comment token to: #605F5F (neutral.600)
```

**Comment by Diaan** [comment]
Changing token.property to accent.500 will make it too close to --highlight-color (both blue) so I changed it to danger.500. I also changed some other token colours to values from the SL pallette:


![Image](https://github.com/sl-design-system/components/assets/1351956/6234a4d8-13b3-47a4-ac29-5b8bbdd69db8)
(you don't want the red and blue to be the same colour, what would happen with accent.500)


![Image](https://github.com/sl-design-system/components/assets/1351956/f69f26a8-b793-44fa-8702-ce11963da4e8)
(uses success.500)

**Comment by Diaan** [comment]
The SL theme file is already loaded in the website (otherwise we couldn't show the examples) so all the tokens are available. (so in that sense the task is complete)

The designs don't use the SL theme tokens in a consistent way so it wouldn't make much sense to apply the tokens that are there now and then do it again when the designs have been revised to use only SL theme tokens.

**Comment by Diaan** [comment]
Fixed in #1006 (linking doesn't work)
